EvidenceApi/serverless.yml

```diff
@@ -2,7 +2,7 @@ service: evidence-api
 variablesResolutionMode: 20210326
 provider:
   name: aws
-  runtime: dotnet6
+  runtime: dot8
```

Comment:
dot8

Reply:
dot dot dot
thanks 🙏 addressed in [2680a58](https://github.com/LBHackney-IT/evidence-api/pull/234/commits/2680a58f5d584ef345196e767f33c3a0d73ab928)